core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceV2.scala

```diff
@@ -67,20 +67,35 @@ trait WorkspaceApiServiceV2 extends UserInfoDirectives {
                 }
             }
         } ~
-          path("bucketMigration") {
-            post {
-              entity(as[List[WorkspaceName]]) { workspaceNames =>
-                complete {
-                  bucketMigrationServiceConstructor(ctx)
-                    .migrateAllWorkspaceBuckets(workspaceNames)
-                    .map(StatusCodes.Created -> _)
+          pathPrefix("bucketMigration") {
```

Comment:
Didn't you already fix this?

Reply:
It's a different group of endpoints. That was to fix everything under `api/workspaces/v2/{workspaceNamespace}/{workspaceName}/...` and this let's us add other endpoints under `api/workspaces/v2/bucketMigration/...`